TGS.ControlPanel/ControlPanel/ControlPanel.Designer.cs

```diff
@@ -1061,7 +1061,7 @@ private void InitializeComponent()
 			// 
 			// BYONDPanel
 			// 
-			this.BYONDPanel.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(39)))), ((int)(((byte)(40)))), ((int)(((byte)(34)))));
+			this.BYONDPanel.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(133)))), ((int)(((byte)(143)))), ((int)(((byte)(160)))));
```

Comment:
mm, you changed a ton of these

Reply:
Yes, if I understand the code correctly that should change the background color and make it a lighter grey to allow for easier viewing of the buttons